packages/parse5/lib/common/html.ts

```diff
@@ -1,4 +1,5 @@
-export enum NAMESPACES {
+/** All valid namespaces in HTML. */
+export enum NS {
```

Comment:
We always aliased `NAMESPACES` as `NS`, even in this file. This makes it the actual name.